README.md

```diff
@@ -309,6 +317,20 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
   - One way to prevent DDoS is to check state validation when user submits the transaction (before tx goes to pending pool)
   - Node must handle state check validation part while accepting transactions. If any node decides to skip this implementation, other nodes will ignore all future transactions from particular node.
 
+17. Exit
+
+  - Q. Why Matic doesn't have Exit despite of it is like Plasma?
+  - A. Exit is not necessarily needed for sidechain. It is one of security model to achieve customer asset preservation. In our model, Delegates reselection ceremony and fast chain resuming are the security model for assets.
```

Comment:
Matic will have similar Exit but now we will provide liquidity for small fees who wants to exit instantly.

Reply:
Ref
https://github.com/maticnetwork/whitepaper/pull/4/files#r195947256